.pfnci/config.pbtxt

```diff
@@ -433,6 +433,41 @@ configs {
   }
 }
 
+# BENCHMARK + HEAD
+configs {
+  key: "cupy.linux.benchmark"
+  value {
+    requirement {
+      cpu: 8
+      memory: 50
+      disk: 10
+      gpu: 1
+    }
+    time_limit: {
+      seconds: 21600
+    }
+    environment_variables { key: "GPU" value: "1" }
+    command: ".pfnci/linux/main-flexci.sh benchmark"
+  }
+}
+
+configs {
+  key: "cupy.linux.benchmark.pr"
+  value {
+    requirement {
+      cpu: 8
+      memory: 50
+      disk: 10
+      gpu: 1
+    }
+    time_limit: {
+      seconds: 21600
+    }
+    environment_variables { key: "GPU" value: "1" }
+    command: ".pfnci/linux/main-flexci.sh benchmark"
+  }
+}
+
```

Comment:
This can be removed?

Reply:
I will be using this config for PRs, master vs released branch will use the other config and run daily